jujugui/static/gui/src/app/components/charmbrowser/test-charmbrowser.js

```diff
@@ -43,14 +43,12 @@ describe('Charmbrowser', function() {
         }
       }};
     var changeState = sinon.stub();
-    var addService = sinon.stub();
     var charmstoreSearch = sinon.stub();
     var output = jsTestUtils.shallowRender(
       <juju.components.Charmbrowser
         appState={appState}
         changeState={changeState}
-        charmstoreSearch={charmstoreSearch}
-        addService={addService} />);
```

Comment:
Well, remember that you can just bind the app at the top level when you're passing in the `addService` method, so no need to pass app around (that's the whole point). We're not using `addService` anywhere at the moment so feel free to modify it as you see fit.